### PR TITLE
Rename behats multishop features for easier to follow convention

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Just a little naming change, so its easier to follow the convention by adding the "multishop" suffix at the end of feature, instead of mixing it somewhere in the middle.
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI passing will be enough as it purely only change for behat tests
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

File: tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/update_multishop_combination_stock.feature [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-multi-shop-combination-stock
```
Comment (jolelievre) [inline tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/update_multishop_combination_stock.feature:1]: The file should be renamed to update_combination_stock_multishop.feature no?

File: tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-multi-shop-management
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags multishop_management
```
Comment (jolelievre) [inline tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature]:
```suggestion
# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags shop-management
```
If we follow the implied convention from other changes the tag should match the file name, so either we adapt this tag or we should rename the file itself to multishop_management.feature I guess

File: tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 @restore-products-before-feature
 @clear-cache-before-feature
 @restore-shops-after-feature
 @restore-languages-after-feature
 @reset-img-after-feature
 @clear-cache-after-feature
-@product-multi-shop
-@update-multi-shop-management
+@product-multishop
+@multishop_management
```
Comment (jolelievre) [inline tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature]:
```suggestion
@multishop-management
```
We don't use any underscore in other tags so we should keep it that ay

**Review by jolelievre — COMMENTED** [review]
Thanks @zuk3975 

Just a few comments and it's good to me

**Review by jolelievre — APPROVED** [review]
Thanks @zuk3975

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Tests are green, I'm merging it. Thanks @zuk3975 .

**Pull request merged**